components/Calendar/Calendar.js

```diff
@@ -0,0 +1,142 @@
+// #FFFFFF - white
+// #F5F5F5 - off-white
+// #B0B4D4 - light purp
+// #84C7D0 - sea blue
+// #4BD2D2 - ugly blue
+
+
+import { useState, useEffect } from 'react';
+// import { Scheduler, WeekView, DayView, Appointments, Toolbar, DateNavigator, ViewSwitcher,}
+//import moment from 'moment';
+import { styled, alpha } from '@mui/material/styles';
+import Paper from '@mui/material/Paper';
+import { ViewState } from '@devexpress/dx-react-scheduler';
+import {
+  Scheduler,
+  WeekView,
+  Toolbar,
+  DateNavigator,
+  Appointments,
+  TodayButton,
+  AppointmentTooltip,
+} from '@devexpress/dx-react-scheduler-material-ui';
+import TooltipContent from './subcomponents/TooltipContent.js';
+import styles from '../../utils/styles/CalendarStyles/Calendar.module.css';
+
+
+const data = [
+  {
+    title: 'KEvin CLASS',
+    startDate: new Date(2022, 2, 21, 6, 35),
+    endDate: new Date(2022, 2, 21, 22, 30),
+
+    zoomlink: 'http://garbase.com'
+  },
+  {
+  title: 'Other Class',
+  startDate: new Date(2022, 2, 24, 6, 35),
+  endDate: new Date(2022, 2, 24, 10, 30),
+  zoomlink: 'http://assaffsaafs.com'
+},]
+
+
+const PREFIX = 'Pog';
+
+const classes = {
+  todayCell: `${PREFIX}-todayCell`,
+  dayCell: `${PREFIX}-dayCell`,
+  todayHeader: `${PREFIX}-today`,
+  dayHeader: `${PREFIX}-weekday`,
+};
+
+const StyledWeekViewTimeTableCell = styled(WeekView.TimeTableCell)(({ theme }) => ({
```

Comment:
Since we are basically assigning this new "styled" component to the variable StyledWeekViewTimeTableCell, we can move all this styling logic to a base components folder. In that base components folder we can export these out to where we need them. This is so we don't have to go through 80 ish lines of code before we hit the component that the file is named for. If you have questions on this DM me for more explanation.